src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -138,6 +145,124 @@ public static DatasetModel populateDatasetModelFromDataset(
+                          .name("Gender identity")
+                          .kind(SnapshotBuilderProgramDataOption.KindEnum.LIST)
+                          .values(
+                              List.of(
+                                  new SnapshotBuilderListOption().name("FEMALE").id(22),
+                                  new SnapshotBuilderListOption().name("MALE").id(23),
+                                  new SnapshotBuilderListOption().name("NON BINARY").id(24),
+                                  new SnapshotBuilderListOption().name("GENDERQUEER").id(25),
+                                  new SnapshotBuilderListOption().name("TWO SPIRIT").id(26),
+                                  new SnapshotBuilderListOption().name("AGENDER").id(27),
+                                  new SnapshotBuilderListOption()
+                                      .name("No Matching Concept")
+                                      .id(0))),
+                      new SnapshotBuilderProgramDataOption()
+                          .id(4)
+                          .name("Race")
+                          .kind(SnapshotBuilderProgramDataOption.KindEnum.LIST)
+                          .values(
+                              List.of(
+                                  new SnapshotBuilderListOption()
+                                      .name("American Indian or Alaska Native")
+                                      .id(28),
+                                  new SnapshotBuilderListOption().name("Asian").id(29),
+                                  new SnapshotBuilderListOption().name("Black").id(30),
+                                  new SnapshotBuilderListOption().name("White").id(31)))))
+              .featureValueGroups(
+                  List.of(
+                      new SnapshotBuilderFeatureValueGroup()
+                          .id(0)
+                          .name("Condition")
+                          .values(List.of("Condition Column 1", "Condition Column 2")),
+                      new SnapshotBuilderFeatureValueGroup()
+                          .id(1)
+                          .name("Observation")
+                          .values(List.of("Observation Column 1", "Observation Column 2")),
+                      new SnapshotBuilderFeatureValueGroup()
+                          .id(2)
+                          .name("Procedure")
+                          .values(List.of("Procedure Column 1", "Procedure Column 2")),
+                      new SnapshotBuilderFeatureValueGroup()
+                          .id(3)
+                          .name("Surveys")
+                          .values(List.of("Surveys Column 1", "Surveys Column 2")),
+                      new SnapshotBuilderFeatureValueGroup()
+                          .id(4)
+                          .name("Person")
+                          .values(List.of("Demographics Column 1", "Demographics Column 2"))))
+              .prepackagedConceptSets(
```

Comment:
I decided that these would make sense to be configurable by dataset, despite not being in the stub version we have in the UI, so added them.